lib/findJsModulesFor.js

```diff
@@ -48,6 +49,13 @@ function findJsModulesFromModuleFinder(
             hasNamedExports: !isDefault,
           });
         }
+
+        // Filter out modules that are in the `excludes` config.
+        if (config.get('excludes').some((glob: string): boolean =>
+          minimatch(path, glob))) {
```

Comment:
nit: I'm finding the indentation here to be a little confusing. It might be better to break this up in a different way--e.g. storing something in a variable).